src/Components/Modal/WebModal.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { StyleSheet, TouchableOpacity, View } from "react-native";
+import { ModalProps } from "react-native-modal";
+import getShadowStyles from "src/Themes/getShadowStyle";
+import Colors from "src/Themes/Colors";
+
+class WebModal extends React.PureComponent<ModalProps> {
+  private documentBody = document.getElementsByTagName("body")[0];
+  private domElement = document.createElement("div");
+  static defaultProps = {
+    style: {
+      padding: "2%",
+    },
+    backdropColor: Colors.shadow,
+  };
+  constructor(props: ModalProps) {
+    super(props);
+  }
+  public componentDidMount = () => {
+    this.documentBody.appendChild(this.domElement);
+    this.handleModalShow();
+  };
+
+  public componentDidUpdate = (prevProps: ModalProps) => {
+    if (this.props.isVisible && prevProps.isVisible !== this.props.isVisible) {
```

Comment:
There is ` if (!props.isVisible) return null;` in the parent component

So this component will never be rendered, when `isVisible` is false.

If you will change `isVisible` to true, then to false, and then to true, `WebModal` component will be remounted.

So there is no need in this code.